static/downloadables/calculate_score.py

```diff
@@ -165,8 +167,11 @@ def vcfcalculations(snpSet, vcfObj, tableObjDict, isJson, isCondensedFormat, omi
                                             unmatchedAlleleVariants.add(rsID)
 
                 if not omitUnusedStudiesFile and len(oddsRatios) == 0 and len(protectiveVariants) == 0 and len(riskVariants) == 0 and len(unmatchedAlleleVariants) == 0 and len(clumpedVariants) == 0:
+                    samp_count -= 1
```

Comment:
I'm unsure about this. Technically, being in this loop doesn't change the sample count does it?

Reply:
Oh you're right. I've changed it so that the if statements for condensed/not condensed/json options are in their own if/elif blocks. This way the sample count keeps going if a sample falls into this first if statement on line 169. 
I'm confused though... why do we print to the unused trait/study file here? Don't we only print to this file if the study doesn't match any of the samples' variants?

Reply:
I put it here I think because if this if statement is true, we don't want to go into the other if statements because it will try to access stuff that isn't there. And I think if we get to this point with one sample, then it will happen for all the samples, due to the nature of the VCF's. Does that make sense? I can explain it better if you want.

Reply:
Okay, that makes sense. So, when I run it how the code was written originally, I get an error because for one study, the first sample goes into the first if statement (the one with the printunusedstudy stuff).  The second study doesn't go into that same if statement, and instead goes into the 'is condensedFormat' if statement block. Since the samp_count is now at 2 for this sample, the newline doesn't get initiated and it throws an error.

Reply:
I don't quite follow the last sentence

Reply:
Okay, so I'm not getting the same error as I was before... But I no longer have access to the lsprs server, so I'm not using the same test file. Something was happening on line 227. The newline for the condensed output is initiated only for the first sample. For subsequent samples, the prs value is appended to the existing newline. However, for some reason the newline wasn't being initiated (meaning the sample count was never equal to 1 inside the 'elif iscondensed' block) whenever the code entered the if statement on line 169 (if not omit UnusedStudiesFile and len(oddsRatios)==0 etc.). But I can't recreate that error right now, so maybe it's fine now!